src/@types.ts

```diff
@@ -1,20 +1,63 @@
-export interface RawSchema {
+import { Resolver, ForSchemaOptions } from 'avsc'
+
+export enum SchemaType {
+  AVRO = 'AVRO',
+  JSON = 'JSON',
+  PROTOBUF = 'PROTOBUF',
+  UNKNOWN = 'UNKNOWN',
+}
+
+export interface SchemaHelper {
+  validate(schema: Schema): void
+  getSubject(confluentSchema: ConfluentSchema, schema: Schema, separator: string): ConfluentSubject
+}
+
+export type AvroOptions = Partial<ForSchemaOptions>
+export type JsonOptions = any // FIXME:
```

Comment:
This should be `ConstructorParameters<typeof Ajv>[0]` (which I think is `InstanceOptions` from Ajv)